documentation/keepalived_vrrp_sync_group.md

```diff
@@ -0,0 +1,41 @@
+[back to resource list](https://github.com/sous-chefs/keepalived#resources)
```

Comment:
We probably want a list of resources under resources here

Reply:
That will get done as part of the final merge to master (This is going to a staging branch for now)